src/client/app/page.jsx

```diff
@@ -15,12 +15,12 @@ const Footer = styled.footer`
   height: ${footerHeight};
 `;
 
-export default function App() {
+const Page = ({children, navBar}) => {
   return (
     <div>
```

Comment:
Also, we can change `div` to `section` tag and remove all other `section` wrappers from individual component pages. So that we don't render another unnecessary `div`wrapper.